src/widgets/modalChildren/NewConfirmModal.js

```diff
@@ -72,7 +72,7 @@ export const NewConfirmModal = props => {
           )}
         </View>
       </View>
-    </>
+    </ModalContainer>
```

Comment:
We don't want to use `ModalContainer` here.

For example in `DataTablePageModal` - `NewConfirmModal` is a child of `NewModalContainer`. Doing it this way will mean we have a `ModalContainer` within a `ModalContainer`.

Reply:
@joshxg 
What component do you recommend me using more explicitly please?

Reply:
No component - as it was previously using a fragment `<>` - then when we use this component we use 

```
<ModalContainer>
    <NewConfirmModal ... />
</ModalContainer>
```

Reply:
@joshxg 
![Screen Shot 2020-02-17 at 2 02 48 PM](https://user-images.githubusercontent.com/57238075/74616691-413dda80-518e-11ea-97b0-40dc429c9ed2.png)
What do you recon to apply after ConfirmForm? I assume that you have no prob. of changing the name of NewConfirmModal!?

Reply:
as below - I don't see where anyone has suggested `ConfirmForm` as a name, only to remove `New` from `NewConfirmModal` which would make it `ConfirmModal`

Reply:
@joshxg 
<img width="611" alt="Screen Shot 2020-02-18 at 9 27 16 AM" src="https://user-images.githubusercontent.com/57238075/74684529-0a6cd080-5231-11ea-8844-aec7c364f30b.png">
This structure at FinaliseModal deactivate clostButton. What alteration to close the modal please?

Reply:
`ModalContainer` also has props : https://github.com/openmsupply/mobile/blob/70887879aeb32d6ae319ba0f2784e5876b42b28e/src/widgets/modals/ModalContainer.js#L92-L99

You can see here: If you do not pass an `onClose` function, then the close button does not get rendered https://github.com/openmsupply/mobile/blob/70887879aeb32d6ae319ba0f2784e5876b42b28e/src/widgets/modals/ModalContainer.js#L60-L61

Reply:
@joshxg 
Appreciate for comments and help as above.